offchain/rollups-events/src/rollups_outputs.rs

```diff
@@ -12,12 +12,31 @@ decl_broker_stream!(RollupsOutputsStream, RollupsOutput, "rollups-outputs");
 /// Cartesi  output
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub enum RollupsOutput {
+    AdvanceCompletion(RollupsAdvanceCompletion),
     Voucher(RollupsVoucher),
     Notice(RollupsNotice),
     Report(RollupsReport),
     Proof(RollupsProof),
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub struct RollupsAdvanceCompletion {
```

Comment:
The names are not the same because they have different semantics.

- CompletionStatus is the status of the advance step once it is completed. It doesn't have the unprocessed state.
- AdvanceCompletion is the aggregate of the completion status and the input that generated that status. I couldn't think of a better name for this one.
- InputStatus can be either unprocessed or one of the completion statuses.

Reply:
I think I will rename everything just to CompletionStatus; it might be easier to understand.

Reply:
Done at b60f6d6b